.travis.linux.trusty.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+URL=ftp://ftp.simplesystems.org/pub/libpng/png/src/libpng16/libpng-1.6.23.tar.gz
+FILE=libpng-1.6.23.tar.gz
```

Comment:
Yeah; I'm only proposing the submodule as a fallback for systems that don't have libpng available otherwise. It's very similar to what you have here but in a way that works off of Travis too.